sql/core/src/main/scala/org/apache/spark/sql/execution/adaptive/DynamicJoinSelection.scala

```diff
@@ -50,10 +51,16 @@ object DynamicJoinSelection extends Rule[LogicalPlan] {
       mapStats.bytesByPartitionId.forall(_ <= maxShuffledHashJoinLocalMapThreshold)
   }
 
-  private def selectJoinStrategy(plan: LogicalPlan): Option[JoinStrategyHint] = plan match {
+  private def selectJoinStrategy(plan: LogicalPlan,
+                                 joinType: JoinType): Option[JoinStrategyHint] = plan match {
```

Comment:
[style] indentation is a bit off 
can ref this : [scala style guide](https://github.com/databricks/scala-style-guide#spacing-and-indentation)

quoting the piece which suggests what should be done in this case : 

**_For method declarations, use 4 space indentation for their parameters and put each in each line when the parameters don't fit in two lines. Return types can be either on the same line as the last parameter, or start a new line with 2 space indent._**